package-lock.json

```diff
@@ -24,7 +24,7 @@
         "beachball": "^2.31.0",
         "cross-env": "^7.0.3",
         "patch-package": "^6.4.7",
-        "playwright": "1.40.0"
+        "playwright": "^1.30.0"
```

Comment:
@jattasNI this seems wrong for renovate to update. This declaration is for the contents of the package.json of the root file at the time the lock file was created. 

https://github.com/ni/nimble/blob/69edd302cdd22366ed09f3491709cf22ac0c397c/package.json#L50

Seems like a bug for lock file update only mode for this to be changing.

Reply:
@jattasNI resolving this comment to merge the PR but we should decide if we want to file a bug or not. In this case it was resolved by pinning everywhere consistently in the repo so it no longer detects an upgrade is possible. But for a repo where a package may not be the same version throughout the monorepo there is a bug in rennovate's algorithm.